tests/unit/backend/helpers/jest-db.ts

```diff
@@ -198,7 +198,7 @@ const insertEncryptForm = async ({
     admin: user._id,
     responseMode: ResponseMode.Encrypt,
     _id: formId,
-    publicKey: 'publicKey',
+    publicKey: 'vuUYOfkrC7eiyqZ1OCZhMcjAvMQ7R4Z4zzDWB+og4G4=',
```

Comment:
might need to dismiss any gitguardian errors etc for this hehe

Reply:
I think it would've alerted already if there was a problem!